inc/api.php

```diff
@@ -98,13 +179,30 @@ function prepare_post_items( array $items ) : array {
  * @return string         The filtered Juicer post message.
  */
 function get_item_content( string $message, $item ) : string {
+	$posted_time         = maybe_humanize_time( strtotime( $item->external_created_at ) );
+	$social_profile_name = ( $item->poster_display_name ) ? esc_html( $item->poster_display_name ) : esc_html( $item->poster_name );
+	$social_post_src     = $item->source->source;
+	$external_src_link   = $item->external;
+	$link_text           = get_read_more_text( $external_src_link, $social_post_src, $social_profile_name );
+	$link_url            = $external_src_link;
+
+	// Get message from post.
 	$content = wp_kses( make_clickable( $message ), allowed_html() );
+	// Search content for links.
 	preg_match( '/<a ?.*>(.*)<\/a>/', $content, $link_matches );
 
-	if ( $item->external === $link_matches[1] ) {
-		$link_text = __( 'Read More', 'hm-juicer' );
-		$link_url  = $link_matches[1];
-		$content   = str_replace( "<a href=\"$link_url\">$link_url</a>", "<a href=\"$link_url\" class=\"juicer-post__sharing-link\">$link_text</a>", $content );
+	$full_read_more_link = "<a href=\"$link_url\" class=\"juicer-post__sharing-link\" aria-label=\"$link_text, posted $posted_time on $social_post_src\">$link_text <i class=\"fas fa-chevron-right\" aria-hidden=\"true\"></i></a>";
```

Comment:
Aria label should be i18nized

```suggestion
	// Translators: 1: The link text, 2: the time the link was posted, 3: the social network the link was posted to.
	$aria_label = sprintf( __( '%1$s, posted %2ds on %3$s', 'hm-juicer' ), $link_text, $posted_time, $social_post_src );
	$full_read_more_link = "<a href=\"$link_url\" class=\"juicer-post__sharing-link\" aria-label=\"$aria_label">$link_text <i class=\"fas fa-chevron-right\" aria-hidden=\"true\"></i></a>";
```

Reply:
Fixed in [1021254](https://github.com/humanmade/hm-juicer/pull/6/commits/1021254e6a720b596ee55f1fdb7b0dde2f580ca7)